corporate/views/support.py

```diff
@@ -608,6 +614,28 @@ def remote_servers_support(
             support_view_request = SupportViewRequest(
                 support_type=SupportType.delete_fixed_price_next_plan,
             )
+        elif remote_server_status:
+            assert remote_server is not None
+            remote_server_status_billing_session = RemoteServerBillingSession(
+                support_staff=acting_user, remote_server=remote_server
+            )
+            if remote_server_status == "active":
+                do_reactivate_remote_server(remote_server)
```

Comment:
This should probably be reworked in a follow-up to have audit logs for who did it.